src/app-layout/visual-refresh/context.tsx

```diff
@@ -541,17 +541,10 @@ export const AppLayoutInternalsProvider = React.forwardRef(
       function handleSplitPanelMaxWidth() {
         const contentGapRight = 80; // Approximately 40px when rendered but doubled for safety
         const toolsFormOffsetWidth = 160; // Approximately 80px when rendered but doubled for safety
-        const toolsOffsetWidth = isToolsOpen ? toolsWidth : 0;
-        const activeDrawerOffsetWidth = activeDrawerId ? drawerSize : 0;
+        const panelOffsetWidth = isToolsOpen ? toolsWidth : activeDrawerId ? drawerSize : 0;
```

Comment:
Tools width should be accounted only when drawers are not used

```suggestion
        const panelOffsetWidth = drawers ? activeDrawerId ? drawerSize : 0 ? isToolsOpen ? toolsWidth : 0;
```

(or refactor this to if-else, because it is getting too clunky)